tests/server/server-executeBulkWrite-004.phpt

```diff
@@ -28,11 +28,11 @@ foreach ($writeConcerns as $wc) {
 ?>
 ===DONE===
 <?php exit(0); ?>
---EXPECT--
+--EXPECTF--
 OK: Got MongoDB\Driver\Exception\RuntimeException
-not master
+not (primary|master)
 OK: Got MongoDB\Driver\Exception\RuntimeException
-not master
+not (primary|master)
 OK: Got MongoDB\Driver\Exception\RuntimeException
-not master
+not (primary|master)
```

Comment:
Using a regex within [EXPECTF](https://qa.php.net/phpt_details.php#expectf_section) requires wrapping the pattern with `%r`. I think this might be responsible for the CI test failures.

Reply:
Oh. I assumed they could be added directly, so I didn't bother to run them. Thanks!